cmd/flags.go

```diff
@@ -206,3 +229,14 @@ func EnvOrDefault(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func defaultPath(placeholderPath, platformAPI, layersDir string) string {
+	fileName := filepath.Base(placeholderPath)
+	path := filepath.Join(layersDir, fileName) // starting from platform api 0.5, the default directory is the layers dir.
+	if (api.MustParse(platformAPI).Compare(api.MustParse("0.5")) < 0) || (layersDir == "") {
+		// prior to platform api 0.5, the default directory was the working dir.
+		// layersDir is unset when this call comes from the rebaser - will be fixed as part of https://github.com/buildpacks/spec/issues/156
+		path = filepath.Join(".", fileName)
+	}
+	return path
```

Comment:
When possible, I prefer to have one `return` statement from a function (or have:
```
if (api.MustParse(platformAPI).Compare(api.MustParse("0.5")) < 0) || (layersDir == "") {
  return filepath.Join(".", fileName)
} else {
  return filepath.Join(layersDir, fileName)
}
```
but the linter doesn't allow this).

But if the convention is:
```
if (api.MustParse(platformAPI).Compare(api.MustParse("0.5")) < 0) || (layersDir == "") {
  return filepath.Join(".", fileName)
}
return filepath.Join(layersDir, fileName)
```
I can make this change.

Reply:
The latter is the typical convention.

Reply:
Haha, no problem. I'll change it.